README.md

```diff
@@ -879,6 +879,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [FreeCodeCamp](https://www.freecodecamp.org/) - Free and Open Source Career Development. Certificates in Data Analysis, InfoSec, Web Development, and more.
   * [Ladderly.io](https://ladderly.io/) - Free and Open Source Career Development. Includes a social network and job search tools.
   * [The Odin Project](https://www.theodinproject.com/) - Free and Open Source Career Development. Focus on JavaScript and Ruby.
+  * [Redis University](https://university.redis.com/) - Free and Open Source Career Development. Specialize in Redis with Certification
```

Comment:
being focussed on one technology only, I dont really think it fits here.  I am sure there's like a "awesome redis" or some similar list where it would fit better.